src/vs/editor/browser/viewParts/viewCursors/viewCursor.ts

```diff
@@ -105,6 +107,7 @@ export class ViewCursor {
 		}
 		if (e.viewInfo) {
 			this._cursorStyle = this._context.configuration.editor.viewInfo.cursorStyle;
+			this._lineCursorWidth = Math.min(this._context.configuration.editor.viewInfo.lineCursorWidth, this._typicalHalfwidthCharacterWidth);
```

Comment:
`this._typicalHalfwidthCharacterWidth` might change in next block, which makes `this._lineCursorWidth` out of date.

Reply:
Done